vllm/model_executor/models/llama.py

```diff
@@ -271,14 +271,13 @@ def forward(
 @support_torch_compile
 class LlamaModel(nn.Module):
 
-    def __init__(
-        self,
-        config: LlamaConfig,
-        cache_config: Optional[CacheConfig] = None,
-        quant_config: Optional[QuantizationConfig] = None,
-        lora_config: Optional[LoRAConfig] = None,
-        prefix: str = "",
-    ) -> None:
+    def __init__(self,
```

Comment:
I know this is the linter acting up, but can we try to commit without this formatting change?